sources/tw/tao/regionwide.json

```diff
@@ -0,0 +1,23 @@
+{
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "TW-TAO",
+            "country": "Taiwan"
+        },
+        "country": "tw"
+    },
+    "data": "http://gener.tycg.gov.tw/arcgis/rest/services/map/MapServer/18",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": "HOUSE_NO",
+        "street": {
+            "function": "join",
+            "fields": ["STREET","SECT","LANE","ALLEY"],
+            "separator": ""
+        },
+        "city": "LI",
+        "district": "TOWNNAME",
```

Comment:
TOWNNAME is the city, LI is district (technically it's a city district, but OpenAddresses currently doesn't have a field for that, so it's usually stored in district)